types/components/BreadcrumbItem.d.ts

```diff
@@ -8,6 +8,7 @@ export interface BreadcrumbItemProps {
   linkAs?: React.ElementType;
   target?: string;
   title?: React.ReactNode;
+  linkProps?: React.LinkHTMLAttributes<any>;
```

Comment:
Is there a better generic that we can use here other than `any`?

Reply:
should be okay now. thx